examples/nlp/lstm_generator_cityname.py

```diff
@@ -1,19 +1,21 @@
 from __future__ import absolute_import, division, print_function
 
 import os
-from six.moves import urllib
+from six import moves
+import ssl
 
 import tflearn
 from tflearn.data_utils import *
 
 path = "US_Cities.txt"
 if not os.path.isfile(path):
-    urllib.request.urlretrieve("https://raw.githubusercontent.com/tflearn/tflearn.github.io/master/resources/US_Cities.txt", path)
+    context = ssl._create_unverified_context()
+    moves.urllib.request.urlretrieve("https://raw.githubusercontent.com/tflearn/tflearn.github.io/master/resources/US_Cities.txt", path, context=context)
 
 maxlen = 20
 
 X, Y, char_idx = \
-    textfile_to_semi_redundant_sequences(path, seq_maxlen=maxlen, redun_step=3)
+    textfile_to_semi_redundant_sequences(path, seq_maxlen=maxlen, redun_step=3, to_lower_case=True)
```

Comment:
Please remove to_lower_case-True, as it is interesting to see that the model can learn by itself to use upper-case letters in a city name (Usually the first letter).